frontend/src/router/index.js

```diff
@@ -58,6 +61,11 @@ export default new Router({
                 name: 'ReviewList',
                 component: ReviewList
             },
+            {
+                path: '/user-list/add',
```

Comment:
We have user/lists route, the convention is ok, so `users/lists/add` a bit better.